jest.config.js

```diff
@@ -39,8 +39,9 @@ module.exports = {
   ],
   testEnvironment: 'node',
   testPathIgnorePatterns: [
+    'test/unit/repository',
```

Comment:
I've ignored these tests as they should really be refactored into local integration tests and hit the db rather than mocking everything out. However, I don't want to do that as part of this PR. I think it should be done as a future refactoring piece.